servers/rendering/renderer_rd/forward_mobile/scene_shader_forward_mobile.cpp

```diff
@@ -293,7 +295,7 @@ void SceneShaderForwardMobile::ShaderData::set_code(const String &p_code) {
 
 					if (k == SHADER_VERSION_COLOR_PASS || k == SHADER_VERSION_COLOR_PASS_MULTIVIEW || k == SHADER_VERSION_LIGHTMAP_COLOR_PASS || k == SHADER_VERSION_LIGHTMAP_COLOR_PASS_MULTIVIEW) {
 						blend_state = blend_state_blend;
-						if (depth_draw == DEPTH_DRAW_OPAQUE) {
+						if (depth_draw == DEPTH_DRAW_OPAQUE && uses_alpha_clip == false) {
```

Comment:
Looks like we may need this in the clustered code too. Otherwise in the clustered renderer alpha scissor materials will have depth drawing forced disabled.

Reply:
Comparing the code right now. Based on my current understanding my clustered fix should not work. Since it does I am obviously still missing something, and I really don't want to change something I don't understand.

Reply:
Here is the relevant piece of code in the clustered renderer:

https://github.com/godotengine/godot/blob/5c7971d8560cacc7efa06fc35eb90080dcaea9a8/servers/rendering/renderer_rd/forward_clustered/scene_shader_forward_clustered.cpp#L307-L318